packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -219,6 +219,8 @@ export type BlockNoteEditorOptions<
   setIdAttribute?: boolean;
 
   dropCursor?: (opts: any) => Plugin;
+
+  tabKeyboardNavigation?: boolean;
```

Comment:
What do you think of this?

```typescript
/***
Select desired behavior when pressing Tab (or Shift-Tab). Specifically, what should happen when a user has selected multiple blocks, the Formatting Toolbar is open, and presses `Tab`.
- `"prefer-navigate-ui"`: when any toolbars are open, change focus to the toolbar. The user needs to first press `Escape` to close the toolbar, and can then indent multiple blocks. Better for keyboard accessibility.
- `"prefer-indent"`: regardless of whether toolbars are open, indent the selection of blocks. In this case, it's not possible to access the toolbar with the keyboard.

@default "prefer-navigate-ui" because it's
*/
tabBehavior: "prefer-navigate-ui" | "prefer-indent" = "prefer-navigate-ui";
```